### ISSUE TITLE
Precise game rating not accessible (for screen reader and keyboard users)

### ISSUE THREAD

**Issue by eladnarra** [itchio/itch.io#1376]
When you hover over the star rating of a game, it shows a more precise rating. Example: on [irori](https://pomepomelo.itch.io/irori)'s page, they have 5 stars visually, but if you hover over the stars it says their calculated rating is 4.8 (as of writing this comment).

**For keyboard users:** this more precise rating information is inaccessible if you're not using a mouse, because there's no way to focus the rating and interact with it to show the number.

**For screen reader users:** the stars are hidden with aria-hidden="true" (and are created with CSS, so I think they probably wouldn't be picked up by a screen reader anyway?). This is kind of fixed with the inclusion of screenreader only text, but it doesn't state the actual rating - it just says "Rated 5.0 out of 5 stars." It does look like 5 full stars on the page, so maybe it's technically correct, but a screen reader user is still missing out on information a sighted person could get using mouseover.

Somewhat related, the number of reviews may be kind of confusing when using a screenreader - maybe it'd make sense to use screenreader only text here, too? Something like:

`(996<span class="screenreader_only"> reviews</span>)`

**Comment by leafo** [comment]
Good catch, it looks like some rounding was taking place by accident. I've pushed out a fix so the accurate rating is used to display stars and screenreader text.

**Comment by eladnarra** [comment]
Awesome! I guess I should make the keyboard accessibility a different issue?
